src/NuGet.Core/NuGet.Configuration/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,9 @@
 NuGet.Configuration.PackageSourceMapping.PackageSourceMapping(System.Collections.Generic.IReadOnlyDictionary<string, System.Collections.Generic.IReadOnlyList<string>> patterns) -> void
-NuGet.Configuration.NuGetConfigurationException.NuGetConfigurationException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) -> void
\ No newline at end of file
+NuGet.Configuration.NuGetConfigurationException.NuGetConfigurationException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) -> void
+NuGet.Configuration.PackageSourceMapping.Patterns.get -> System.Collections.Generic.IReadOnlyDictionary<string, System.Collections.Generic.IReadOnlyList<string>>
+NuGet.Configuration.PackageSourceMappingProvider
+NuGet.Configuration.PackageSourceMappingProvider.AddOrUpdatePackageSourceMappingSourceItem(NuGet.Configuration.PackageSourceMappingSourceItem packageSourceMappingSourceItem) -> void
+NuGet.Configuration.PackageSourceMappingProvider.GetPackageSourceMappingItems() -> System.Collections.Generic.IReadOnlyList<NuGet.Configuration.PackageSourceMappingSourceItem>
+NuGet.Configuration.PackageSourceMappingProvider.PackageSourceMappingProvider(NuGet.Configuration.ISettings settings) -> void
+NuGet.Configuration.PackageSourceMappingProvider.Remove(System.Collections.Generic.IReadOnlyList<NuGet.Configuration.PackageSourceMappingSourceItem> packageSourceMappingSourceItems) -> void
```

Comment:
A bunch of these methods were only labeled as `public` because the class was internal.

Not everything needs to be public, only what we're going to use/what makes the API complete

Reply:
Why this one is marked as resolved? 
Please make line 19, 30, 50 methods `internal`. 
It'll reduce new methods to 1 instead of 4.

Reply:
Ah that makes sense. I made those three methods internal